asapdiscovery-data/asapdiscovery/data/tests/test_ligand_comparison.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+from asapdiscovery.data.schema_v2.ligand import ChemicalRelationship, Ligand
+
+
+@pytest.fixture(scope="session")
+def base_ligand():
+    return Ligand.from_smiles(
+        compound_name="EDG-MED-5d232de5-3",
+        smiles="CC(=O)N1CC[C@H](c2c1ccc(c2)Cl)C(=O)Nc3cncc4c3cccc4",
+    )
+
+
+@pytest.fixture(scope="session")
+def different_name():
+    return Ligand.from_smiles(
+        compound_name="EDG-MED-5d232de5-3_with_a_different_name",
+        smiles="CC(=O)N1CC[C@H](c2c1ccc(c2)Cl)C(=O)Nc3cncc4c3cccc4",
+    )
+
+
+@pytest.fixture(scope="session")
+def stereoisomer():
+    return Ligand.from_smiles(
+        compound_name="EDG-MED-5d232de5-4",
+        smiles="CC(=O)N1CC[C@@H](c2c1ccc(c2)Cl)C(=O)Nc3cncc4c3cccc4",
+    )
+
+
+@pytest.fixture(scope="session")
+def tautomer():
+    return Ligand.from_smiles(
+        compound_name="EDG-MED-5d232de5-3_tautomer",
+        smiles=r"CC(N1CC[C@@H](/C(O)=N\c2c3ccccc3cnc2)c4cc(Cl)ccc41)=O",
+    )
+
+
+@pytest.fixture(scope="session")
+def acid():
+    return Ligand.from_smiles(
+        compound_name="EDG-MED-5d232de5-4_acid",
+        smiles="CC(N1CC[C@@H](C([NH2+]c2c3ccccc3cnc2)=O)c4cc(Cl)ccc41)=O",
+    )
+
+
+@pytest.fixture(scope="session")
+def other():
+    return Ligand.from_smiles(
+        compound_name="other",
+        smiles="CCCCCC",
+    )
+
+
```

Comment:
Need to also test `is_tautomer`, `is_stereoisomer`, `is_chemically_equal` and `has_defined_stereo` etc.